src/actions/user/passwordActions.js

```diff
@@ -0,0 +1,39 @@
+import { toast } from 'react-toastify';
```

Comment:
assess cognitive complexity in this file

Reply:
The best way will be to use the new api middleware Jonathan introduced so I am waiting for his PR to be merged and I applied the changes.

Reply:
Changes applied.